test/e2e/config/cloudstack.yaml

```diff
@@ -125,6 +126,7 @@ variables:
   CLOUDSTACK_INVALID_CONTROL_PLANE_MACHINE_OFFERING: "OfferingXXXX"
   CLOUDSTACK_EXTREMELY_LARGE_CONTROL_PLANE_MACHINE_OFFERING: "Extremely Large Instance"
```

Comment:
Should the offering name be more descriptive of the compute resource being modeled?

Reply:
Would it be better to name it "impossible offering" ? It is misleading unless the user reads the test details

Reply:
That's a good name for it.  I'll rename it.

Reply:
That change is done and tested.